### ISSUE TITLE
Update to @ember-decorators/babel-transforms ^3.1.0

### ISSUE THREAD

**Issue by Windvis** [miguelcobain/ember-yeti-table#13]
I recently updated my npm dependencies and the following warning started showing after building the app:

```
WARNING: The 'ember-yeti-table' addon is using an older version of @ember-decorators/babel-transforms that uses stage 1 decorators. Please update to @ember-decorators/babel-transforms@v3.1 to remove your dependency on legacy decorators.
WARNING: The 'ember-yeti-table' addon is using an older version of ember-decorators that uses stage 1 decorators. Please update to ember-decorators@v3.1 to remove your dependency on legacy decorators.
```

**Comment by bgantzler** [comment]
This is weird, the latest version of ember-decorators is 3.1.5 no problem, and will be updated in this pull request https://travis-ci.org/miguelcobain/ember-yeti-table/jobs/477658956, but the most recent version of babel-transforms is 2.1.2 so there isn't a 3.1 perhaps it a poorly worded error and the pull request above will fix them both

**Comment by cah-briangantzler** [comment]
So locally the second warning has gone away, but it is still requesting that @ember-decorators/babel-transforms be updated to 3.1

**Comment by cah-briangantzler** [comment]
Ok after a little more reading I think I understand whats going on.

The pull request listed will fix one of the warnings. Fixing the other warning requires a lot more effort as the transforms are targeting an as yet unsupported version of ember-cli-babale (according to the warnings I was getting), and I will leave that to a more dedicated pull request to the problem.

**Comment by miguelcobain** [comment]
Should be fixed on `0.1.0`.
